AdventureClient.py

```diff
@@ -102,7 +102,7 @@ def _set_message(self, msg: str, msg_id: int):
     def on_package(self, cmd: str, args: dict):
         if cmd == 'Connected':
             self.locations_array = None
-            if Utils.get_options()["adventure_options"].get("death_link", False):
+            if Utils.get_settings()["adventure_options"].get("death_link", False):
```

Comment:
I'm guessing it's not worth trying to change this `adventure_options` to `adventure_settings` because that would break people's existing host.yaml files?